ui/src/components/ServiceAccounts/ServiceAccounts.scss

```
@@ -0,0 +1,133 @@
+@import "../../styles/var/colors.scss";
```

Stylelint (17.13.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)